tutorials/01_introductory/lifecycle.py

```diff
@@ -0,0 +1,266 @@
+"""
+=======================
+The Lifecycle of a Plot
+=======================
+
+Going from start to finish of one plot in Matplotlib.
+
+This tutorial aims to show the beginning, middle, and end of a single
+visualization using Matplotlib. We'll begin with some raw data, and
```

Comment:
don't need comma: We'll begin with some raw data and end with an image file containing our customized visualization.